### DISCUSSION TITLE
Multiple binaries in single deb package

### DISCUSSION THREAD

**Discussion by mattcarp12** [goreleaser/goreleaser#2968] [Q&A] [answered]
I need to build multiple binaries and put into a single deb package. Is that possible with goreleaser?

**Comment by caarlos0** [comment, marked as answer]
yes, its the default behavior to package all binaries in the same package, unless you specify an `builds` filter.

**Reply by caarlos0** [reply]
I just tested it with:

<img width="395" alt="image" src="https://user-images.githubusercontent.com/245435/157575956-f4c090df-e018-464f-b1f1-55a00db204eb.png">

and it works as expected:

<img width="685" alt="image" src="https://user-images.githubusercontent.com/245435/157575981-fca7d937-9529-42a6-9128-8e540026c0c7.png">

---

note: please reply in the thread

**Reply by mattcarp12** [reply]
Got it. Thanks for clarifying.

**Comment by mattcarp12** [comment]
If I have multiple builds (for multiple binaries), the behavior I observe is a distinct package is built for each binary, not that each binary is included in a single deb package.
